nvi-rest/src/main/java/no/sikt/nva/nvi/rest/UpdateNviPeriodHandler.java

```diff
@@ -0,0 +1,47 @@
+package no.sikt.nva.nvi.rest;
+
+import static no.sikt.nva.nvi.common.utils.RequestUtil.getUsername;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import no.sikt.nva.nvi.common.model.business.NviPeriod;
+import no.sikt.nva.nvi.common.model.business.NviPeriod.Builder;
+import no.sikt.nva.nvi.common.service.NviService;
+import no.sikt.nva.nvi.common.utils.RequestUtil;
+import no.sikt.nva.nvi.rest.model.NviPeriodDto;
+import no.sikt.nva.nvi.rest.utils.ExceptionMapper;
+import nva.commons.apigateway.AccessRight;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+
+
+public class UpdateNviPeriodHandler extends ApiGatewayHandler<NviPeriodDto, NviPeriodDto> {
+
+    private final NviService nviService;
+
+    public UpdateNviPeriodHandler(NviService nviService) {
```

Comment:
Does a handler need a no param constructor?

Reply:
It does, but for now we do not have defaultService to use. I add default constructor with null service